src/Umbraco.Web.UI.Client/src/views/propertyeditors/multiurlpicker/multiurlpicker.controller.js

```diff
@@ -104,7 +95,7 @@ function multiUrlPickerController($scope, angularHelper, localizationService, en
                     }
 
                     if (link.udi) {
-                        var entityType = link.isMedia ? "media" : "document";
+                        var entityType = model.target.isMedia ? "Media" : "Document";
```

Comment:
We are changing the case here for "Media" vs "media and "Document" vs "document", is this expected since i don't see other changes here that might be dealing with this.

Reply:
There's a comparison https://github.com/umbraco/Umbraco-CMS/blob/c96cd09dba012e87eabb133196ce0fd6f82c3067/src/Umbraco.Web.UI.Client/src/views/propertyeditors/multiurlpicker/multiurlpicker.controller.js#L102 where it uses "Document", could've changed that instead i guess

Reply:
Ok cool, just wondering if we should be same and compare these strings as invariant with (example) `"Hello".localeCompare("hello") >= 0` ?

Reply:
We could use `localeCompare` but in this case I don't think it makes sence since we are comparing with a value we've just set our self.

Also note that `localeCompare` returns `0` if the values are equal (see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare)